src/display.c

```diff
@@ -351,11 +362,17 @@ PyImaging_GrabScreenWin32(PyObject *self, PyObject *args) {
         dpiAwareness = SetThreadDpiAwarenessContext_function((HANDLE)-3);
```

Comment:
We need to match the capturing thread DPI awareness to the target window DPI awareness which can be queried with GetWindowDpiAwarenessContext (also added in Windows 10 version 1607): https://learn.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-getwindowdpiawarenesscontext

Without it I get a black border around applications without native DPI scaling support such as Microsoft Spy++.

Reply:
Ok, I've pushed a commit.